workflows/metatdenovo.nf

```diff
@@ -92,6 +92,7 @@ include { UNPIGZ as UNPIGZ_EUKULELE        } from '../modules/local/unpigz.nf'
 include { UNPIGZ as UNPIGZ_CONTIGS         } from '../modules/local/unpigz.nf'
 include { COLLECT_FEATURECOUNTS            } from '../modules/local/collect_featurecounts.nf'
 include { COLLECT_STATS                    } from '../modules/local/collect_stats.nf'
+include { SUM_EGGNOG                       } from '../modules/local/sum_eggnog.nf'
```

Comment:
There isn't a subworkflow for eggnog? If there is, it would be better to hide this there I think.

Reply:
I can do it!